packages/events/TopLevelEventTypes.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {DOMTopLevelEventType} from 'react-dom/src/events/DOMTopLevelEventTypes';
+
+export type TopLevelType =
+  | DOMTopLevelEventType
+  | 'topMouseDown'
```

Comment:
Can you add a comment clarifying these are RN types? Or even declare them separately as `RNTopLevelEventType` and use that in the union?